src/style.css

```diff
@@ -1,3 +1,86 @@
+* {
+  box-sizing: border-box;
+}
+
```

Comment:
Better not to use * selector, you can set this on those elements you need
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/9cb5b354-5444-4136-98f7-6181b9e7925e)